static/app/views/replays/detail/network/index.tsx

```diff
@@ -69,22 +70,33 @@ function NetworkList({networkSpans, startTimestampMs}: Props) {
       deps,
     });
 
-  const initialRequestDetailsHeight = useMemo(
-    () => Math.max(150, window.innerHeight * 0.25),
-    []
-  );
   const {size: containerSize, ...resizableDrawerProps} = useResizableDrawer({
     direction: 'up',
-    initialSize: initialRequestDetailsHeight,
+    initialSize: Math.max(150, window.innerHeight * 0.4), // cannot depend on containerRef :(
     min: 0,
     onResize: () => {},
   });
   const {getParamValue: getDetailRow, setParamValue: setDetailRow} = useUrlParams(
     'n_detail_row',
     ''
   );
-  const detailItemIndex = getDetailRow();
-  const splitSize = networkSpans && detailItemIndex ? containerSize : undefined;
+  const detailDataIndex = getDetailRow();
+  const detailRowIndex = Number(detailDataIndex) + 1;
+
+  const maxContainerHeight =
+    (containerRef.current?.clientHeight || window.innerHeight) - 90;
```

Comment:
What does the `90` represent here?

Reply:
made a const and renamed to `RESIZEABLE_HANDLE_HEIGHT`